docs/src/modules/java/pages/actions-publishing-subscribing.adoc

```diff
@@ -32,6 +35,8 @@ include::example$java-eventsourced-counter/src/main/proto/com/example/actions/co
 
 The class `CounterJournalToTopicAction` gets generated for us based on the proto file defined above.
 
+ifdef::todo[TODO: EventSourced counter example Scala]
```

Comment:
The way I did it in the views doc was to add a Scala tab, but inside it just have `TODO`. Might be confusing to only have Java tab? Not important though.

see #512

Reply:
ok I'll do the same no worries.